students/eowyn/windrevenue/windrevenue/parse_met_data.py

```diff
@@ -0,0 +1,43 @@
+#!/usr/bin/env python3
+
+""" 
+Met data handling for Backcast
+
+Read observational meteorological data
+Determine the highest wind speed sensor and extract data
+Drop other columns
+Replace missing values with NaN
+Drop rows with any NaN
+Resample time series to hourly
+"""
+
+import pandas as pd
+import numpy as np
+
+class parse_met_data():
+
+    #  Given a pandas series, return series with outliers replaced by nan
+    #  For series with few unique variables, do nothing and return the series
+    #  The hepatitis dataset has many columns with 2 or 3 unique variables
+    def replace_outliers_with_na(x):
```

Comment:
no self? is this supposed to be a @staticmethod ?

And if so, maybe a stand alone function is fine.